### ISSUE TITLE
Update SDK Version Once Mutable IAVL is Supported

### ISSUE THREAD

**Issue by alexanderbez** [cosmos/ethermint-archive#465]
`LoadVersion` is extremely slow and the [PR #1952](https://github.com/cosmos/cosmos-sdk/pull/1952/files) in the SDK addresses this by using new IAVL updates. Just a friendly reminder that once that is merged into the SDK and released, update the SDK version in Ethermint to that version -- this will help greatly will loading the initial state.

**Comment by alexanderbez** [comment]
The release is coming soon!
